repo/blob/azure/azure_versioned_test.go

```diff
@@ -19,6 +19,46 @@ import (
 	"github.com/kopia/kopia/repo/format"
 )
 
+func TestGetBlobVersionsFailsWhenVersioningDisabled(t *testing.T) {
```

Comment:
Test seems to have passed
https://github.com/kopia/kopia/actions/runs/7294187986